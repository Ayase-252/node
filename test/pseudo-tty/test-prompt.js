'use strict';

const common = require('../common');
const assert = require('assert');

(async () => {
  const resp1 = prompt('what your favorite food?');
  assert.strictEqual(resp1, 'sushi');

  const resp2 = prompt('what is the default value?', 'DEFAULT');
  assert.strictEqual(resp2, 'DEFAULT');
})().then(common.mustCall(() => {}));
