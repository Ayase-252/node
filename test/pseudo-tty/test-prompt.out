sushi
what your favorite food?
what is the default value?
